examples/pony/window_chain/window_chain.pony

```diff
@@ -0,0 +1,242 @@
+/*
```

Comment:
This app doesn't actually make sense in its current version.  If we keep it, we should modify it so that it's meaningful.